src/main/atm2lndType.F90

```diff
@@ -563,6 +567,11 @@ subroutine InitHistory(this, bounds)
          avgflag='A', long_name='atmospheric partial pressure of CO2', &
          ptr_lnd=this%forc_pco2_grc)
 
+    this%forc_po3_grc(begg:endg) = spval
+    call hist_addfld1d (fname='PO3', units='Pa',  &
+         avgflag='A', long_name='atmospheric partial pressure of O3', &
+         ptr_lnd=this%forc_po3_grc)
```

Comment:
From the perspective of future coupling to the atmosphere, it would probably make sense to go with your first suggestion - activate it only if ozone is taken from the atmosphere.